api/src/main/resources/underlays/aou_synthetic.yaml

```diff
@@ -340,7 +340,7 @@ relationships:
 datasets:
   - name: aou_synthetic
     bigQueryDataset:
-      projectId: broad-tanagra-dev
+      projectId: verily-tanagra-dev
```

Comment:
I suspect you'll need to regenerate the SQL goldenfile tests, I see a bunch of references

Reply:
Yep, done.